cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -101,6 +102,7 @@
         extends AbstractOperator<T, P, S, CrdOperator<C, T, L>> {
 
     private static final Logger log = LogManager.getLogger(AbstractConnectOperator.class.getName());
+    private static final LoggerWrapper loggerWrapper = new LoggerWrapper(log);
```

Comment:
This will be confusing. People will tend to keep using `log` instead of `loggerWrapper. Does it make more sense to do something like this?

```
    private static final LoggerWrapper LOG = new LoggerWrapper(LogManager.getLogger(AbstractConnectOperator.class.getName()));
```

Is there any reason this would not work?

Reply:
`loggerWrapper` is used only for the logs which use `reconciliation` object (this object carries the marker). So removing the `log` is not a way.

Reply:
But shouldn't ideally all of them use it?

Reply:
Ideally yes.